kinder/pkg/cluster/status/cri.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package status
+
+import (
+	"github.com/google/uuid"
+	"github.com/pkg/errors"
+
+	kinddocker "sigs.k8s.io/kind/pkg/container/docker"
+	kindexec "sigs.k8s.io/kind/pkg/exec"
+)
+
+// NB. code implemented in this package ideally should be in the CRI package, but ATM it is
+// implemented here to avoid circular references. TODO: refactor
```

Comment:
introducing sub packages should solve this.

Reply:
yes, but I will address this in a follow up PR